docs/man-pages.xml

```diff
@@ -0,0 +1,36 @@
+<!-- Copyright (c) 2019-2022, see AUTHORS. Licensed under MIT License, see LICENSE. -->
+<reference xmlns="http://docbook.org/ns/docbook"
+           xmlns:xlink="http://www.w3.org/1999/xlink"
+           xmlns:xi="http://www.w3.org/2001/XInclude">
+ <title>Nix-on-Droid Reference Pages</title>
```

Comment:
is it going anywhere?

Reply:
But as mentioned in the PR description
> a `documentation.manPages.enable` option to install the man page on device.

We could install it on the device for easy access to option docs.